### PR TITLE
RunTime support, modular folder support, Money System, and improved command gen for easy MySQL table creation

### PR DESCRIPTION
**Pull request by ZippyMagic**
Complete money integration, rewards, better pack prices, generators for easier table creation, and modules for easier scripting

#3 MERGED INTO THIS

- [x] Add lottery system
- [x] Fix undefined bug
- [x] Add MoneyManager module
- [x] Add MySQL Table generator
- [x] Add `daily` module for daily reward
- [x] Add daily reward command

Final review
- [x] Bug fix
- [x] Recode repetitions
- [x] Update MySQL and finish

Example excerpt for Money Per Message RunTime
```js
const MoneyManager = require('../modules/MoneyManager.js');
class Money extends MoneyManager {}
const money = new Money();

module.exports.run = async (client, message, args, settings, con) => {
    money.establish(con, message.author.id);

    let money_gained = parseInt(money.get()) + (parseInt(message.content.length) / 10) < 100 ? parseInt(money.get()) + (parseInt(message.content.length) / 10): 0;

    money.update(money_gained)
}
```

### CODE REVIEW

**Comment by ZippyMagician** [comment]
If you add more, I will consider adding it

File: lib/commands/buypack.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports.run = async (client, message, args, settings, con) => {
         if(rows.length < 1) return message.channel.send("Please create an account first with `c.create (USERNAME)`!");
         
         let money = parseInt(rows[0].money);
-        if(money >= 100) {
+        if(money >= 150) {
```
Comment (ZippyMagician) [inline lib/commands/buypack.js]: Price may be too much, if you add in some way to GET money I will consider it

File: lib/commands/lottery.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    let s3 = lottery.data[num3];
+    
+    var money_gained = s1 === s2 && s2 === s3 ? (parseInt(args[0]) * 10): (parseInt(args[0]) * -1);
+    
+    con.query(`INSERT INTO userdata ('${message.author.id}', '${data.username}', '${data.feature}', '${data.money + money_gained}');
+    
+    msg.delete();
+    if(money_gained.toString().startsWith("-")) return message.channel.send("You did not win!") else return message.channel.send("You won! Congrats!");
+    
+    await message.channel.send(`\`\`\`${lottery.data[parseInt(num1) - 1].emoji} ${lottery.data[parseInt(num2) - 1].emoji} ${lottery.data[parseInt(num3) - 1].emoji}\n${s1.emoji} ${s2.emoji} ${s3.emoji}\n${lottery.data[parseInt(num1) + 1].emoji} ${lottery.data[parseInt(num2) + 1].emoji} ${lottery.data[parseInt(num3) + 1].emoji}\`\`\``)
```
Comment (ZippyMagician) [inline lib/commands/lottery.js]: If hit with a resulting number of 1 OR max, undefined will be shown. Fix this

**Comment by ZippyMagic** [comment]
I fixed all of the stuff you asked

**Comment by ZippyMagician** [comment]
Looks good, finish the final part on your checklist and it will look good. If you have any more suggestions for 2.0.0, then just create more pulls and I(or Card-Bot) will review them.

**Comment by ZippyMagic** [comment]
Added a MoneyManager module for easier support, also `modules` folder in root `src` so you can add some if you want. I recommend(maybe) some SQL support module, but that is just an idea

**Comment by ZippyMagic** [comment]
Lol the gist of this pull request has changed so much

**Comment by ZippyMagic** [comment]
Went from a simple change to an entire money system and then some

**Comment by ZippyMagic** [comment]
RunTime added

**Comment by ZippyMagic** [comment]
This is a very large Pull Request

**Comment by ZippyMagician** [comment]
It is indeed. This basically IS version 2.0

**Comment by ZippyMagician** [comment]
Actually, I have not planned anything. After this branch is merged, I will work on some commands. Feel free to do some as well. I need trading and global market

**Comment by ZippyMagic** [comment]
Added `daily` and FINISHED!!!

**Comment by ZippyMagician** [comment]
Great!

**Pull request merged**